OpenRA.Mods.Common/Scripting/Global/MapGlobal.cs

```diff
@@ -108,7 +108,7 @@ public string TerrainType(CPos cell)
 		}
 
 		[Desc("Returns true if there is only one human player.")]
-		public bool IsSinglePlayer { get { return Context.World.LobbyInfo.IsSinglePlayer; } }
+		public bool IsSinglePlayer { get { return Context.World.LobbyInfo.NonBotPlayers.Count() == 1; } }
```

Comment:
Although this is in line with the description: Do we really want to count bot-only matches as non-singleplayer? Imho we should rather adjust the description and use `NonBotPlayers.Count() == 1`.